docs/tutorial/kymotracking.rst

```diff
@@ -164,6 +164,24 @@ are simply dropped from the trace.
 There is also an optional keyword argument `initial_sigma` that can be used to pass an initial guess for :math:`\sigma` in micrometers
 in the above expectation equation to the optimizer. The default value is `1.1*pixel_size`.
 
+When lines are well separated, it is possible to use a relatively large window and estimate the peak parameters and offset from the fit directly.
+When this is not the case, one can estimate the offset separately.
+To do this, crop an area of the Kymograph that only has background in it.
+Here we crop the kymograph from 14 to 15 seconds and 9 to 14.5 microns::
+
+    kymo_cropped = kymo["14s":"15s"].crop_by_distance(9, 14.5)
+
+Computing the appropriate photons/pixel background considering a Poissonian noise model can be done by computing the mean of the pixels in this area.
+
+    offset = np.mean(kymo_cropped.green_image)
```

Comment:
this code isn't formatted properly, need the `::` at the end of line 174

Reply:
Fixed! 👍